controls/cis_rhel8.yml

```
@@ -1747,7 +1747,9 @@ controls:
     levels:
       - l2_server
       - l2_workstation
-    status: manual
+    status: automated
```

```suggestion
    status: manual
```
We should keep the status aligned to the requirement, in case of CIS, even if we have a rule for this. In any case, I believe it is safe to keep this rule listed there since it will just install the package. The `4.2.2.1.2` is more complicated to have an automated rule.

---

@marcusburghardt  I think you will want to check 4.2.1.6 or other rules that have Manual in the title but status is automated, there are quite a few

---

Correct. It should also be updated to `status: manual` in `4.2.1.6`. Thanks for noticing this. I didn't find other cases like this. Would you mind to update in this PR? Otherwise I can create a separate one.

---

If that's okay, I will leave that to you in another PR with any CCE changes you need.

---

just dropped the rhel bits altogether from this PR.